tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -0,0 +1,183 @@
+
+    To use the AWS CLI to create an AWS Interface Endpoint, perform the following steps:
+
+    1. Install AWS Command Line Interface (AWS CLI).
+
+        ```bash
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+        unzip awscliv2.zip
+        sudo ./aws/install
+        ```
+
+    2. Configure AWS CLI according to your account information. To get the information required by AWS CLI, see [AWS CLI configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+
+        ```bash
+        aws configure
+        ```
+
+    3. Copy the command in the **Create VPC Interface Endpoint** area and run it in your terminal to create the VPC interface endpoint.
+
+    > **Tip:**
+    >
+    > - If your service is spanning across more than three availability zones (AZs), an error is returned. To resolve the issue, contact [PingCAP Technical Support](https://docs.pingcap.com/tidbcloud/tidb-cloud-support).
+    >
+    >     In addition to the AZs where your TiDB cluster is located, if there is an extra AZ in your selected region, this issue will occur.
+    >
+    > - You cannot copy the command until TiDB Cloud finishes creating endpoint service in the background.
+
+    </div>
+    </SimpleTab>
+
+    Then click **Next**.
+
+#### Step 5: Accept the endpoint connection
+
+After you have created an endpoint service, you are at the **Accept Endpoint Connection** stage. Fill in the box with the your VPC endpoint ID and click **Next**.
+
+You can get your VPC endpoint ID from your AWS Management Console. If you do not know how to get it, click **Show Instruction** and you will see a snapshot of the AWS Management Console that illustrates how to get it. To fold the snapshot, click the **Hide instruction**.
+
+#### Step 6: Enable Private DNS
+
+After you have accepted the endpoint connection, you are at the **Enable Private DNS** stage. Click the **Copy** button to copy the command and run it in your AWS CLI. The `<your_vpc_endpoint_id>` placeholder is automatically replaced with the value you have provided in Step 5.
+
+Then click **Create** to finalize the creation of the private endpoint.
+
+### Connect to a private endpoint service
+
+Take the following steps to connect to a private endpoint service using a private endpoint:
+
+1. In the TiDB Cloud console, navigate to the **Active Clusters** page and click the name of your newly created cluster.
+2. Click **Connect**. The **Connect to TiDB** dialog box is displayed.
+3. Select the **Private Endpoint** tab. If you have created a private endpoint, it is displayed under **Step 1: Create Private Endpoint**.
+4. Under **Step 2: Connect your application**, click the tab of your preferred connection method, and then connect to your cluster with the connection string. The placeholders `<cluster_endpoint_name>:<port>` in the connection string are automatically replaced with the real values.
+
+## Billing information
```

Comment:
remove billing info to https://docs.pingcap.com/tidbcloud/tidb-cloud-billing#data-transfer-cost

Reply:
done